pkg/agent/openflow/client.go

```diff
@@ -1406,6 +1426,17 @@ func (c *client) SendIGMPRemoteReportPacketOut(
 	return c.bridge.SendPacketOut(packetOutObj)
 }
 
+func (c *client) InstallMulticastForwardInboundGroup(groupID binding.GroupIDType) error {
+	c.replayMutex.RLock()
+	defer c.replayMutex.RUnlock()
+
+	group := c.featureMulticast.multicastForwardInboundGroup(groupID, MulticastRoutingTable.GetID())
```

Comment:
Add comment to explain why directly resubmit the packet to "MulticastRoutingTable" rather than going into the first table in multicast pipeline?

Reply:
updated. I think it should be the first table of the multicast pipeline. No need to bypass egress tables.